src/core/math_func.hpp

```diff
@@ -346,6 +346,23 @@ static inline int RoundDivSU(int a, uint b)
 	}
 }
 
+/**
+ * Computes (a / b) rounded away from zero.
+ * @param a Numerator
+ * @param b Denominator
+ * @return Quotient, rounded away from zero
+ */
+static inline uint DivAwayFromZero(int a, uint b)
```

Comment:
the fact that `a` is an int, but `b` is a uint, which is then immediately cast to an int, is very weird. Especially with the uint return value...

Reply:
Oops sorry, I must have blindly copied from the CeilDiv function.  The return type should have been `int`.  But the denominator only makes sense when non-negative, so judging by the `RoundDivSU` function immediately above, using `uint` for the parameter `b` seems more consistent.